persist/migrations/000004_cache.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE public.cached_responses (
+    id bigserial primary key,
+	url varchar NOT NULL,
+	body bytea NOT NULL,  
+	expires_at bigint NOT NULL
+);
+
+CREATE INDEX cached_responses_url_idx ON public.cached_responses (url);
+CREATE UNIQUE INDEX cached_responses_url_uk ON public.cached_responses (url);
```

Comment:
Wouldn't the second index suffice?

Reply:
You're right, removed [f126d6f](https://github.com/breez/breez-lnurl/pull/22/commits/f126d6f9384e1497eee557f0269f4eb907916292)